pybossa.js

```diff
@@ -54,13 +54,25 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     }
 
     function _saveTaskRun(taskrun) {
-        return $.ajax({
-            type: 'POST',
-            url: url + 'api/taskrun',
-            dataType: 'json',
-            contentType: 'application/json',
-            data: taskrun
-        });
+        if(window.pybossa.isGoldMode){
+            const projectId = JSON.parse(taskrun).project_id;
+            return $.ajax({
+                type: 'POST',
+                url: url + `api/project/${projectId}/taskgold`,
```

Comment:
Looks like the only thing that changes between the requests is the url, maybe the if could just set that

Reply:
String template might not work on all browsers

Reply:
agree